auto/sendfile

```diff
@@ -16,11 +16,15 @@ NXT_HAVE_HPUX_SENDFILE=NO
 nxt_feature="Linux sendfile()"
 nxt_feature_name=NXT_HAVE_LINUX_SENDFILE
 nxt_feature_test="#include <sys/sendfile.h>
+                  #include <sys/types.h>
+                  #include <sys/socket.h>
 
                   int main() {
                       off_t  offset;
 
                       sendfile(-1, -1, &offset, 0);
+                      send(-1, NULL, 0, MSG_MORE);
```

Comment:
This test will be passing again as soon as Solaris will get MSG_MORE support. In general it's not a good idea to workaround the test with checking something unrelated.

It's better to reorder the tests. We want to use sendfilev() where it's available, then FreeBSD sendfile() and only then Linux (as it has the less powerful interface).

Reply:
Ok, I thought it was the other way around as linux sendfile is tested first.